packages/webapp/src/pages/Connection/Authorization.tsx

```diff
@@ -137,7 +137,12 @@ export default function Authorization(props: AuthorizationProps) {
             {(connection.credentials.type === 'OAUTH2' || connection.credentials.type === 'APP') && connection.credentials.access_token && (
                 <div className="flex flex-col">
                     <span className="text-gray-400 text-xs uppercase mb-1">Access Token</span>
-                    <SecretInput disabled value={refreshing ? 'Refreshing...' : connection.credentials.access_token} copy={true} refresh={handleForceRefresh} />
+                    <SecretInput
+                        disabled
+                        value={refreshing ? 'Refreshing...' : connection.credentials.access_token}
+                        copy={true}
```

Comment:
Is it true that no other types besides oauth2 have a refresh token? I don’t recall exactly

Reply:
this input is only displayed for Oauth2 and App, and it doesn't seems to contain refresh_token